universitat-vechta-katholische-theologie-de-DE.csl

```diff
@@ -0,0 +1,347 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" et-al-use-first="3" name-as-sort-order="all" demote-non-dropping-particle="sort-only" default-locale="de-DE">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Universitaet Vechta - Katholische Theologie</title>
```

Comment:
Please add your language into the title. You can use the Umlaut also.
E.g.: Albert-Ludwigs-Universität Freiburg - Geschichte (Deutsch)